### DISCUSSION TITLE
Error importing ROSCO_toolbox.inputs

### DISCUSSION THREAD

**Discussion by daansl1** [NatLabRockies/ROSCO#121] [Q&A] [answered]
Dear ROSCO community,

I am trying to tune my own ROSCO controller for the NREL 5MW reference wind turbine using tune_ROSCO.py, however I am not able to run the script. Every time I run tune_ROSCO.py,  an error is returned that ROSCO_toolbox.inputs does not exist. Specifically from the line: 
`from ROSCO_toolbox.inputs.validation import load_rosco_yaml`
the error:
ModuleNotFoundError: No module named 'ROSCO_toolbox.inputs'
is returned.
If I first activate the rosco environment and then run, I get a different error:
ModuleNotFoundError: No module named 'wisdem.inputs'
I have installed and tested WISDEM successfully and installed ROSCO according to the installation instructions on https://rosco.readthedocs.io/en/latest/source/install.html#full-rosco. The same happens if I run python in an Anaconda powershell prompt and import ROSCO.

Perhaps this is a trivial problem and I might have overlooked something very simple, but as of yet I cannot seem to solve this problem.  Your help is appreciated.

Regards,
Daan

**Comment by nikhar-abbas** [comment, marked as answer]
Hi Daan,

It sounds like something is wrong with your install. When you are in your ROSCO virtual env, do you see WISDEM or the ROSCO toolbox when you do `conda list`?  If not, I would suggest starting fresh with your ROSCO install and keeping an eye out on the outputs to make sure no errors are happening.  

Also, do any of the examples run? I would expect them to also fail given the error, but those are more rigorously tested than tune_rosco.py, so are a better place to start when just checking the install. 

Cheers,
Nikhar

**Reply by dzalkind** [reply]
Hey Daan,

Another user had a similar issue, where an old version of WISDEM was installed using conda.  I'm not sure why this happened.

`conda install wisdem=3.5.0` seemed to fix things.

Dan

**Reply by daansl1** [reply]
Hi Nikhar and Dan,

Thank you both for your answer! Indeed my installation was off and installing the new WISDEM version has solved it for me.  Thanks again!

Daan
